cost-analyzer/values.yaml

```diff
@@ -57,10 +58,14 @@ global:
           window: weekly # or 7d
           aggregation: namespace
           filter: kubecost
-          ownerContact: # ownerContact(s) should be the same for the same namespace, otherwise the last namespace alert overwrites
-            - owner@example.com
-            - owner2@example.com
-
+          # Cost Change Alert
```

Comment:
i'm not a fan of this name. i actually like something like `anomaly` better. do you feel strongly about this current name?

reminder to add to docs once merged!

Reply:
Sorry it was a working title that I stuck with until now haha. I'm not a fan of CostChange either but I worry that something with "Anomaly" would be confusing for international/ESL users. How do you feel about `spendChange` like in the initial feature design doc?

Reply:
I'm on board with that!

Reply:
cool. and sounds good; documentation is the first thing I work on after this gets released!